docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -27,47 +27,47 @@ Scientists from across multiple labs and organizations have volunteered to revie
-    |                  |                                                | and NSSL computers                                                                |
-    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
-
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Ben Blake (@BenjaminBlake-NOAA)                | Output visualization, Rocoto                                                                |
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Ratko Vasic (@RatkoVasic-NOAA)                 | Workflow, NCO requirements, and operational platform testing                                |
+    +------------------+------------------------------------------------+---------------------------------------------------------------------------------------------+
+    | EPIC             | Mark Potts (@mark-a-potts)                     | HPC systems                                                                                 |
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Jong Kim (@jkbk2004)                           | UFS Weather Model configuration, forecast sensitivity analysis, data assimilation           |
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Natalie Perlin (@natalie-perlin)               | Generic Linux/Mac installations, hpc-stack/spack-stack                                      |
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Gillian Petro (@gspetro-NOAA)                  | Documentation                                                                               |
+    |                  +------------------------------------------------+---------------------------------------------------------------------------------------------+
+    |                  | Edward Snyder (@EdwardSnyder-NOAA)             | WE2E testing, input data                                                                    |
+    +------------------+------------------------------------------------+---------------------------------------------------------------------------------------------+
```

Comment:
Please add my name to the list of EPIC Reviewers and Areas of Expertise:

`EPIC Michael Lueken (@MichaelLueken) SRW App code management`

Reply:
Done!

Reply:
Oof, sorry! I already had to fix the "ue" (had written "eu"). On the upside, I now know how to spell your name properly! ;P

Reply:
No worries!  Thank you very much for adding me!